vocab/properties.json

```diff
@@ -22,7 +22,7 @@
         ]
     },
     "accessibility": {
-        "description": null,
+        "description": "Ability to access something.",
```

Comment:
isn't this the level of accessibility (not just true/false) ?

Reply:
@apdavison : that's a good point. I'll change it.